src/pages/partners/payment-agent/index.tsx

```diff
@@ -20,28 +23,33 @@ const meta_attributes = {
 }
 
 const PaymentAgent = () => {
-    return (
-        <Layout type="partners" margin_top={10} no_login_signup>
-            <SEO
-                title={localize('Payment agents | Partnership programmes | Deriv')}
-                description={localize(
-                    'Find out how to become a payment agent on Deriv to expand your client base and earn extra revenue.',
-                )}
-                meta_attributes={meta_attributes}
-            />
-            <Helmet>
-                <script type="application/ld+json">{JSON.stringify(faq_schema)}</script>
-            </Helmet>
-            <Hero />
-            <TapInto />
-            <YourControl />
-            <WhoCanApply />
-            <ROW>
-                <P2PBanner />
-            </ROW>
-            <Faq />
-        </Layout>
-    )
+    const { is_row } = useCountryRule()
+
+    if (is_row) {
+        return (
+            <Layout type="partners" margin_top={10} no_login_signup>
+                <SEO
```

Comment:
The `<SEO>` component should not be any kind of conditionals, because on build phase gatsby has no idea about the value of `is_row` so the `SEO` tags such as `title` would have no values in the page source.

Reply:
Thanks @michio-rework , will do the changes.

Reply:
thanks @meenakshi-deriv :pray: